app/services/waste_exemptions_engine/confirmation_pdf_generator_service.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module WasteExemptionsEngine
+  class ConfirmationPdfGeneratorService < BaseService
+    def run(registration:)
+      @registration = registration
+
+      pdf_generator.pdf
+    end
+
+    private
+
+    def pdf_generator
+      GeneratePdfService.new(template)
+    end
+
+    def template
+      ApplicationController.new.render_to_string(
+        pdf: "certificate",
+        template: "waste_exemptions_engine/pdfs/certificate",
+        encoding: "UTF-8",
+        locals: locals
+      )
+    end
+
+    def locals
+      {
+        presenter: presenter
+      }
+    end
+
+    def presenter
```

Comment:
Same point about one line methods. Is there a need for this rather than calling `CertificatePresenter.new(@registration)` directly?

Reply:
Ditto of https://github.com/DEFRA/waste-exemptions-engine/pull/129/files#r276214828 :D